README.md

```diff
@@ -3,21 +3,21 @@
 
 # KernelSU
 
-A Kernel based root solution for Android devices.
+A Kernel-based root solution for Android devices.
 
 ## Features
 
 1. Kernel-based `su` and root access management.
-2. Module system based on overlayfs.
+2. Module system based on overlays.
```

Comment:
This should be `overlayfs` because it is a term.

Reply:
I apologise for the misconception. I'll fix that.